package-lock.json

```diff
@@ -0,0 +1,8024 @@
+{
```

Comment:
I think this file should not be stored in repository.

Reply:
this file should be stored in repository. Every developers should have one version of this file to check compatibility of packages.